org.libreoffice.LibreOffice.json

```diff
@@ -48,8 +48,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://kerberos.org/dist/krb5/1.16/krb5-1.16.2.tar.gz",
-                    "sha256": "9f721e1fe593c219174740c71de514c7228a97d23eb7be7597b2ae14e487f027"
+                    "url": "https://kerberos.org/dist/krb5/1.20/krb5-1.20.1.tar.gz",
+                    "sha256": "7e022bdd3c851830173f9faaa006a230a0e0fdad4c953e85bff4bf0da036e12f"
```

Comment:
Why would you want to do all these changes here?

The use of external project versions here is exactly the same as in the upstream LO version.  The list here and the upstream list at <https://git.libreoffice.org/core/+/refs/tags/libreoffice-7.5.1.2/download.lst> are kept in sync with a tool from <https://git.libreoffice.org/core/+/c5f620f9f6abae0213fdda17fae10f281b7bdf9f%5E!/>.  In general, it does not work (causing either build or runtime failures) to just randomly replace versions of external proejct sources here.

Reply:
(The above comment should only pertain to the below changes in the `libreoffice` `sources` section, from ant all through zxing-cpp.  But I now see that you meanwhile created yet another <https://github.com/flathub/org.libreoffice.LibreOffice/pull/218> "Update GVFS, Kerberos5, Apache Ant and OpenJDK Flathub Extension", and this whole PR here is probably moot now and should be abandoned?)

Reply:
Sorry for my mistake, i will close this